SilKit/source/CMakeLists.txt

```diff
@@ -243,7 +243,7 @@ if (MSVC)
             PDB_OUTPUT_DIRECTORY_DEBUG ${SILKIT_SYMBOLS_DIR}/Debug
             LINK_FLAGS "/DEBUG" #make sure the resulting .dll has a .pdb file
     )
-elseif(UNIX AND NOT APPLE)
+elseif(UNIX AND NOT APPLE AND NOT (CMAKE_SYSTEM_NAME MATCHES Android))
```

Comment:
Is the build-id also broken? Otherwise I'd only wrap the `target_link_options(SilKit PRIVATE "LINKER:--version-script=${CMAKE_CURRENT_SOURCE_DIR}/silkit.map")` in `if (NOT CMAKE_SYSTEM_NAME MATCHES Android)` (or whatever the correct cmake syntax for that is).

Reply:
Build-ID is working, tested it with a combination of `SILKIT_BUILD_REPRODUCIBLE` and `SILKIT_USE_LLD` options. I'll move the Android check to guard the linker script only.